### ISSUE TITLE
error_running_doclog.

### ISSUE THREAD

**Issue by triska** [aarroyoc/doclog#5]
I followed the steps in the documentation, and also adjusted the configuration file, and now I get:

<pre>
<b>$ bash doclog.sh scryer.config.pl </b>
doclog(1,1,0).
error_running_doclog.
</pre>

**Comment by aarroyoc** [comment]
The output folder must not exist (`make clean`). Also, check if you have the latest dependencies (`make setup`).

**Comment by triska** [comment]
I think one reason is that I have **temporary files** in the `lib` directory, these are created by Emacs when editing files:

<pre>
all:user:path_segments(A,[[],"home","markus","scryer-prolog","src","lib",".","a.pl~"]).
exit:user:path_segments("/home/markus/scryer-prolog/src/lib/./a.pl~",[[],"home","markus","scryer-prolog","src","lib",".","a.pl~"]).
call:user:directory_exists("/home/markus/scryer-prolog/src/lib/./a.pl~").
</pre>

**Comment by triska** [comment]
The same issue for the `.ediprolog` file that exists in the library path:

<pre>
call:user:directory_exists("/home/markus/scryer-prolog/src/lib/./.ediprolog").
</pre>

Maybe `doclog` can be made more robust against such files? I think it cauld suffice to process only `*.pl` files.

**Comment by aarroyoc** [comment]
Yes, those are good points. File handling in DocLog was an afterthought and it can be improved a lot. I'll keep this issue open until I fix it.
